python-test/features/steps/functions.py

```diff
@@ -46,46 +48,20 @@ def new_agent(token, name, tagKey, tagValue, base_orb_url = base_orb_url):
     new_agent = requests.post(base_orb_url+'/api/v1/agents', json= {"name": name,"orb_tags": {tagKey: tagValue}, "validate_only": False}, headers={'Content-type':'application/json',  'Accept':'*/*', 'Authorization': token})
     return new_agent.status_code, json.loads(new_agent.text)
 
-def send_terminal_commands(command, separator=None, cwd_run=None):
-    args = shlex.split(command)
-    docker_command_execute = subprocess.Popen(args, stdout=subprocess.PIPE, cwd=cwd_run)
-    subprocess_return = docker_command_execute.stdout.read().decode()
-    if separator == None:
-        subprocess_return_terminal = subprocess_return.split()
-    else:
-        subprocess_return = subprocess_return.split()
-        subprocess_return_terminal = subprocess_return[0].split(separator)
-    return subprocess_return_terminal[0]
 
 def check_logs(logs):
     match = False
     providing_log_regex = "'level':'info','ts':.+,'caller':'agent/rpc_to.go:.+','msg':'sending capabilities','value':'{.'schema_version.':.+,.'orb_agent.':{.'version.':.+},.'agent_tags.'.+.'backends.':{.'pktvisor.':{.'version.':.+.',.'data.':{.'taps.':{.'default_pcap.':{.'config.':{.'iface.':.+,.'pcap_source.':.+},.'input_type.':.'pcap.',.'interface.':.+.'"
     try:
         for i in logs:
             i_replaced = i.replace('"',"'")
```

Comment:
I'm pretty sure `i` should be valid json. Can you try `logline = json.loads(i)` and see if we can use that interface instead of text matching? Then you can search for match on `logline['msg'] == 'sending capabilities'` for example?